SECURITY.md

```diff
@@ -0,0 +1,37 @@
+# Security Policy
+
+## Supported Versions
+
+Security-related reports are considered for official numbered releases
+starting with v3.5. However, issues that do not affect the current Stable or
+Beta series are unlikely to be fixed. Please see
+http://www.squid-cache.org/Versions/ for the list of releases that belong to
+the current series.
+
+Reports about security issues in the Development series are welcomed. However,
+development series contains experimental code that does not qualify for CVE
+allocation under MITRE policy.
```

Comment:
I failed to find any reference to experimental code in MITRE documentation. Could you please share a link? I would like to double check that we are not officially misinterpreting official MITRE limitations.

Reply:
https://cve.mitre.org/cve/cna/rules.html#section_7-4_requirements_for_assigning_a_cve_id
"7.4.7 CNAs SHOULD NOT assign CVE IDs to vulnerabilities in products that are not publicly available or licensable."

The Miter representative from RHEL who mentored me on the process informed that our formal numbered release packages are what are considered a "product". A vulnerability which is introduced in private branches, or fixed before it got to a numbered release should not have CVE. That includes our snapshot tarballs.

Reply:
Thank you for providing those references. AFAICT, the opinion of the RHEL person does not quite match the published MITRE policy. I suggest removing the words `under MITRE policy` to avoid a misinformation risk without debating whether it exists. I do not insist on this change.

Reply:
```suggestion
Reports about security issues in the Development series are welcomed. However,
development series contains experimental code that does not qualify for CVE
allocation.
```